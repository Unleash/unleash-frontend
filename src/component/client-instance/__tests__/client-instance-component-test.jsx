import React from 'react';

import ClientStrategies from '../client-instance-component';
import renderer from 'react-test-renderer';

jest.mock('react-mdl');

test('renders correctly with no clientInstances', () => {
    const tree = renderer.create(<ClientStrategies fetchClientInstances={jest.fn()} clientInstances={[]} />).toJSON();

    expect(tree).toMatchSnapshot();
});
