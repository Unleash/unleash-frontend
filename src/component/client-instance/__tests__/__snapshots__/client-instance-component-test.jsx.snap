// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly with no clientInstances 1`] = `
<react-mdl-DataTable
  rows={Array []}
  selectable={false}
  style={
    Object {
      "width": "100%",
    }
  }
>
  <react-mdl-TableHeader
    name="instanceId"
  >
    Instance ID
  </react-mdl-TableHeader>
  <react-mdl-TableHeader
    name="appName"
  >
    Application name
  </react-mdl-TableHeader>
  <react-mdl-TableHeader
    name="clientIp"
  >
    IP
  </react-mdl-TableHeader>
  <react-mdl-TableHeader
    name="createdAt"
  >
    Created
  </react-mdl-TableHeader>
  <react-mdl-TableHeader
    name="lastSeen"
  >
    Last Seen
  </react-mdl-TableHeader>
</react-mdl-DataTable>
`;
